tools/formatter/FormatterTool.java

```diff
@@ -96,6 +96,19 @@ public void format(Path path) throws IOException {
     }
   }
 
+  static class BuildFormatter implements BaseFormatter {
+    static BuildFormatter init() {
```

Comment:
Why did we do `init()` instead of an empty constructor?

Reply:
@oferb Indeed! 🤔